src/utils/TypeGuards.ts

```diff
@@ -6,8 +6,8 @@
 
 /* tslint:disable */
 import * as ts from "typescript";
```

Comment:
You can re-generate this file based on your changes by running: `npm run code-generate`.

Sidenote: I'm going to abstract out the duplicate pattern going in in these methods in the next few weeks.